cmd/kube-controller-manager/app/options/options.go

```diff
@@ -103,7 +103,7 @@ func NewCMServer() *CMServer {
 			ConcurrentGCSyncs:        20,
 			ClusterSigningCertFile:   "/etc/kubernetes/ca/ca.pem",
 			ClusterSigningKeyFile:    "/etc/kubernetes/ca/ca.key",
-			ReconcilerSyncLoopPeriod: metav1.Duration{Duration: 5 * time.Second},
+			ReconcilerSyncLoopPeriod: metav1.Duration{Duration: 60 * time.Second},
```

Comment:
The default 5 second also has a problem that - polling an entire cluster one by one (like how we do currently) is going  to take more than 5 seconds even if done parallely, so likely we have overlapping polls going on.